### ISSUE TITLE
When using CocoaPods you are required to use Rosetta simulators

### ISSUE THREAD

**Issue by danigutierrezayuso** [growthbook/growthbook-swift#43]
Right now when adding the Growthbook sdk using CocoaPods the podfile include these lines:

```
 spec.xcconfig                  = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
 spec.pod_target_xcconfig       = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
 spec.user_target_xcconfig      = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

That forces you to use Rosetta simulators instead of the native ones when using an arm Mac.

It would be great to have native arm support in the sdk so we avoid having to use Rosetta in the simulators.

**Comment by alvarogoj** [comment]
Impressive! I hope the GrowthBook team feels the same way. Working with Rosetta simulators leads me to constantly "play" with the Mac's beach ball

**Comment by mattials** [comment]
Please fix this, I'm getting bald from waiting so long for it to compile with Rosetta. 💇

**Comment by danigutierrezayuso** [comment]
Here is the PR: https://github.com/growthbook/growthbook-swift/pull/48

**Comment by danigutierrezayuso** [comment]
This was already fixed in version [1.0.47](https://github.com/growthbook/growthbook-swift/releases/tag/1.0.47)
